.github/workflows/kitchen.windows.yml

```diff
@@ -0,0 +1,121 @@
+---
+name: 'Kitchen (Windows)'
+'on': ['push', 'pull_request']
+
+env:
+  machine_user: kitchen
+  machine_pass: Password1
+  machine_port: 5985
+  KITCHEN_LOCAL_YAML: 'kitchen.windows.yml'
+
+jobs:
+  test-2019:
+    runs-on: 'windows-2019'
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        instance:
+          - py3-stable-3001-windows-2019
+          - py3-stable-3002-windows-2019
+          - latest-windows-2019
+    steps:
+      - name: 'Check out code'
+        uses: 'actions/checkout@v2'
+      - name: 'Install Chef'
```

Comment:
do we need chef?  can we tell kitchen to ignore it?

Reply:
Let's ask @dafyddj -- I'm sure he will be able to give a clear response to this query.

Reply:
yeah, let's go this way for a while, I do really hope GH actions will support win/fbsd one day, at least I've heard a lot of talks about it recently